packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -107,6 +108,26 @@ module.exports = {
     }
   },
 
+  connect: async (ctx, next) => {
+    _.defaultsDeep(strapi.plugins['users-permissions'].config.grant, {
+      server: {
+        protocol: 'http',
+        host: 'localhost:1337'
```

Comment:
The `host` value should be more dynamic using `strapi.config.currentEnvironment.server.host`? Not sure about that but I have a doubt